src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.ts

```diff
@@ -88,16 +93,19 @@ export class EditKeyboardMappingComponent implements OnInit {
     return this.selectedAction.argument.type
   }
 
-  public save(): void {
-    const keyboardMapping: KeyboardMapping = {
-      action: {} as Tv2PartAction,
-      actionTrigger: {
-        ...this.keyboardMapping?.actionTrigger,
-        ...this.actionTriggerForm.value,
+  public save(shouldClose: boolean): void {
```

Comment:
`shouldClose` is a boolean flag which is a bad practice.
Boolean flags indicates that the method has more than one responsibility. If it didn't it wouldn't need the flag.
Instead you should create methods that explicitly states what happens. That could look something like:
```
// Don't do this:
public buildTimeline(withBlueprints: boolen): Timeline {
  ...
}

// Do this instead
public buildTimelineWithBlueprints(): Timeline {
  ...
}

public buildTimelineWithoutBlueprints(): Timeline {
  ...
}
```

Reply:
Fixed